tcpdialer.go

```diff
@@ -326,46 +326,16 @@ func (d *TCPDialer) tryDial(network string, addr *net.TCPAddr, deadline time.Tim
 		}
 	}
 
-	chv := dialResultChanPool.Get()
-	if chv == nil {
-		chv = make(chan dialResult, 1)
-	}
-	ch := chv.(chan dialResult)
-	go func() {
-		var dr dialResult
-		dr.conn, dr.err = net.DialTCP(network, d.LocalAddr, addr)
-		ch <- dr
-		if concurrencyCh != nil {
-			<-concurrencyCh
-		}
```

Comment:
This is still needed after the `DialContext` returns (before the return out of the function).

Reply:
Done